Content.Server/Fluids/EntitySystems/DrainSystem.cs

```diff
@@ -31,6 +126,13 @@ public override void Update(float frameTime)
                 }
                 drain.Accumulator -= drain.DrainFrequency;
 
+                // Disable ambient sound from emptying manually
+                if (!drain.AutoDrain)
+                {
+                    _ambientSoundSystem.SetAmbience(drain.Owner, false);
+                    return;
+                }
```

Comment:
You should split this out into a separate component as you're just re-using draincomponent's dumping behaviour and ignoring the rest of it just for sinks.

Also this return would cause all other drains to stop working.

Reply:
I'm more a fan of having one component that can do both on this one, because ultimately drains and sinks have the same behaviour. It wouldn't be desirable to duplicate the same feature in two very close systems.

I don't get the comment about other drains stopping to work